.appveyor.yml

```diff
@@ -37,7 +37,8 @@ build_script:
 test_script:
   - node --version
   - npm --version
-  - npm run test
+  - npm run test:without-system-tests
```

Comment:
Running the system tests with coverage requires a different routine than normal testing. But since we don't want to run them twice, we split it up here. So we run the non-system tests and then the system tests, with the coverage mechanism.